configure_combinations

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+sh build_posix/reconf
+
+curdir=`pwd`
+
+flags="CFLAGS=\"-Werror -Wall -Wextra -Waddress -Waggregate-return -Wbad-function-cast -Wcast-align -Wdeclaration-after-statement -Wformat-security -Wformat-nonliteral -Wformat=2 -Wmissing-declarations -Wmissing-field-initializers -Wmissing-prototypes -Wnested-externs -Wno-unused-parameter -Wpointer-arith -Wredundant-decls -Wshadow -Wundef -Wunused -Wwrite-strings -O -fno-strict-aliasing -Wuninitialized\"
+CC=clang CFLAGS=\"-Wall -Werror -Qunused-arguments -Wno-self-assign -Wno-parentheses-equality -Wno-array-bounds\""
+
+options="--enable-diagnostic
+--disable-shared
+--disable-static
+--enable-java --enable-python
+--enable-snappy --enable-zlib --enable-lz4
+--with-builtins=lz4,snappy,zlib
+--enable-python
```

Comment:
`--enable-python` itself may not deserve a separate test. Suggest to drop this line.

Reply:
removed